src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClientCertificateManager.cs

```diff
@@ -176,10 +176,13 @@ public static void InstallLocalCertificateFromBridge()
                 }
             }
 
-            InstallCertificateToMyStore(certificateToInstall);
-
-            // We also need to install the root cert if we install a local cert
-            InstallRootCertificateFromBridge();
+            // certificateToInstall could be null if local cert has been installed in the previous tests
+            if (certificateToInstall != null)
+            {
```

Comment:
So maybe we should clarify the comment to say that some tests pass in null?  The phrase "previous test" makes this sound like it might be affected by test order.  NIT only.